Bio-KBase-Auth/lib/Bio/KBase/AuthToken.pm

```diff
@@ -585,6 +592,54 @@ sub go_request {
 
 }
 
+sub _get_token {
+    my $self = shift @_;
+    my %p = @_;
+
+    my $json;
+    eval {
+	my $baseurl = $Bio::KBase::Auth::AuthSvcHost;
```

Comment:
Yes, I know.  I left it for now just so I could do some of my own testing.

I did find out today that there actually is a way to put this parameter in a config file: it comes from the .kbase_config file.  We can leave it in there or I can move it to the deployment.cfg file, I'm not sure it matters.  But I'm not happy with how it currently gets configured, it's a little opaque, so I do want to modify that to be clearer.